### PR TITLE
update pkgdown with dev version

### PR DESCRIPTION
**Pull request by statnmap**
Allow to use the last template for pkgdown available properly

### CODE REVIEW

File: DESCRIPTION [hunk 1]
```diff
@@ -43,6 +43,7 @@ Imports:
     config,
     crayon,
     desc,
+    devtools,
```
Comment (ColinFay) [inline DESCRIPTION]: Can you please remove these dependencies from the DESCRIPTION? 

Thanks :)

**Comment by statnmap** [comment]
{attachment} found a dependency to devtools in your code, this is why it is listed in Depends.  
https://github.com/ThinkR-open/golem/blob/a222c5224d9dee2c4d8ac5635a109f4291cfcf19/R/add_deploy_helpers.R#L563

**Comment by VincentGuyader** [comment]
> 
> 
> {attachment} found a dependency to {devtools} in your code, this is why it is listed in Depends.
> 
> https://github.com/ThinkR-open/golem/blob/a222c5224d9dee2c4d8ac5635a109f4291cfcf19/R/add_deploy_helpers.R#L563

not anymore :)

**Comment by statnmap** [comment]
Changing branch: https://github.com/ThinkR-open/golem/pull/451

**Pull request closed without merging**